recipe/meta.yaml

```diff
@@ -1,32 +1,35 @@
-{% set version = "0.7.3" %}
+{% set version = "0.8.0" %}
 
 package:
   name: hvplot
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/h/hvplot/hvplot-{{ version }}.tar.gz
-  sha256: 74b269c6e118dd6f7d2a4039e91f16a193638f4119b4358dc6dbd58a2e71e432
+  sha256: 8630dba34969b105e267cbe14237e2d56d9969c801980ae5e5190bc18b2792d6
 
 build:
-  number: 1
-  noarch: python
+  number: 0
+  # ModuleNotFoundError: No module named 'hvplot.__main__'
+  # entry_points:
+  #   - hvplot = hvplot.__main__:main
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - python
+    - pyct
```

Comment:
`pyct >=0.4.4`
https://github.com/holoviz/hvplot/blob/v0.8.0/setup.py#L102

Reply:
I've just updated `pyct` https://github.com/AnacondaRecipes/pyct-feedstock/pull/1. There were missing artifacts for v0.4.6 and v0.4.8.

Reply:
I pinned `pyct>=0.4.4`. Done, thanks!